src/types.ts

```diff
@@ -1,6 +1,6 @@
 import type { RequestOptions, OctokitResponse } from "@octokit/types";
 
-export type RequestErrorOptions = {
+export interface RequestErrorOptions extends ErrorOptions {
```

Comment:
tsc complains Cannot find name 'ErrorOptions' 

>       Prettier │ Processed 17 files in 0.41s.
>       tsc      │ ../../node_modules/.pnpm/@octokit+request-error@7.1.0/node_modules/@octokit/request-error/dist-types/types.d.ts(2,46): error TS2304: Cannot find name 'ErrorOptions'.
>       tsc      │ tsc --noEmit exited with code 1
>       ESLint   │ Processed 11 files in 1.37s.

Reply:
Duplicate of #541, #545, #550